packages/gasket-plugin-intl/README.md

```diff
@@ -60,6 +60,7 @@ required. However, these options exist to customize an app's setup.
   - `excludes` - (string[]) List of modules to ignore
 - `nextRouting` - (boolean) Enable [Next.js Routing] when used with
   [@gasket/plugin-nextjs]. (default: true)
+- `preloadLocales` (boolean) - Preloads the locale files from the manifest file in the middlewareHook.
```

Comment:
Nitpicking the description here to keep it more general incase we decide to move the preloading particulars to another lifecycle or something in the future.

```suggestion
- `preloadLocales` (boolean) - Preloads the locale files from the manifest at startup,
  allowing a faster first response.
```

Reply:
done 👍